src/lib/states.ts

```diff
@@ -126,6 +126,21 @@ export class InverterStates {
                 state: {id: 'dailyEnergyYield', name: 'Daily energy yield', type: 'number', unit: 'kWh', role: 'value.energy'},
                 register: {reg: 32114, type: ModbusDatatype.uint32, length: 2, gain: 100}
             },
+            {
```

Comment:
Reg 32114 is already there... see line 127

Reply:
Der 32114 scheint bei mir, habe einen Akku, den Wert wieder zu spiegeln was ich heute benutzt habe und nicht der gesamte ertrag